homeassistant/components/samsungtv/media_player.py

```diff
@@ -61,6 +69,7 @@
     | MediaPlayerEntityFeature.SELECT_SOURCE
     | MediaPlayerEntityFeature.NEXT_TRACK
     | MediaPlayerEntityFeature.TURN_OFF
+    | MediaPlayerEntityFeature.TURN_ON
```

Comment:
Base on this change, I think you can remove lines L217-L129:
https://github.com/home-assistant/core/blob/5d09fe8dc184488e399348877c21cfc9a39ca955/homeassistant/components/samsungtv/media_player.py#L127-L129

Reply:
Done